js/ad_manager_vast.js

```diff
@@ -2123,7 +2123,7 @@ OO.Ads.manager(function(_, $) {
       var mediaFile = linearXml.find("MediaFile");
 
       parseTrackingEvents(result.tracking, linearXml);
-      if (mediaFile.size() > 0) {
+      if (mediaFile.length > 0) {
```

Comment:
The updated jquery no longer supports the size() API. They suggest to use .length instead.

Reply:
Wouldn't this be using the jquery version from the core at runtime, which would still be the old version?

Reply:
This should still be fine. The .length returns the same value as .size(). Both of these APIs have existed since version 1.0 of jquery:

https://api.jquery.com/length/